spec/src/lib.rs

```diff
@@ -221,30 +221,38 @@ impl Genesis {
     fn build_cellbase_transaction(&self) -> Result<Transaction, Box<dyn Error>> {
         let mut outputs =
             Vec::<CellOutput>::with_capacity(1 + self.system_cells.len() + self.issued_cells.len());
+        let mut outputs_data = Vec::with_capacity(outputs.capacity());
 
         // Layout of genesis cellbase:
         // - genesis cell, which contains a message and can never be spent.
         // - system cells, which stores the built-in code blocks.
         // - issued cells
-        outputs.push(self.genesis_cell.build_output()?);
-        self.system_cells.build_outputs_into(&mut outputs)?;
+        let (output, data) = self.genesis_cell.build_output()?;
+        outputs.push(output);
+        outputs_data.push(data);
+        let (system_cells_outputs, system_cells_data) = self.system_cells.build_outputs()?;
+        outputs.extend(system_cells_outputs);
+        outputs_data.extend(system_cells_data);
         outputs.extend(self.issued_cells.iter().map(IssuedCell::build_output));
+        outputs_data.extend(self.issued_cells.iter().map(|_| Bytes::new()));
```

Comment:
So if there are multiple outputs, concat the data together?

Reply:
For `outputs` and `outputs_data`, the length must equal, and for every item in `outputs_data`, `hash(outputs_data[i]) == outputs[i].data_hash`